packages/plugins/opentelemetry/src/plugin-utils.ts

```
@@ -0,0 +1,104 @@
+    return {
+      get() {
+        if (!states[scope]) states[scope] = new WeakMap<any, any>();
+        let value = states[scope].get(key as any);
+        if (!value) states[scope].set(key, (value = {}));
+        return value;
+      },
+      enumerable: true,
+    };
+  }
+
+  let pluginWithState: Record<string, (payload: any) => unknown> = {};
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'pluginWithState' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)